### ISSUE TITLE
Bug in the gradio_UI

### ISSUE THREAD

**Issue by alexrame** [CarperAI/ArchitextRL#3]
Hi, thanks for open-sourcing your code and congrats on the project. Unfortunately, so far I can not make the gradio_UI work. 

With the default query: "house with two bedrooms and one bathroom",
when calling
https://github.com/CarperAI/ArchitextRL/blob/2673c676e7ded74818a79b6372bf7cdfbd012049/gradio_UI.py#L75
it raises
```
  File "site-packages/transformers/models/gptj/modeling_gptj.py", line 232, in forward
    k_rot = apply_rotary_pos_emb(k_rot, sin, cos)
  File "site-packages/transformers/models/gptj/modeling_gptj.py", line 79, in apply_rotary_pos_emb
    return (tensor * cos) + (rotate_every_two(tensor) * sin)
RuntimeError: The size of tensor a (48) must match the size of tensor b (64) at non-singleton dimension 3
```

My setup:
Python: 3.8.16
torch version: 1.13.1+cu117
transformers: 4.28.0.dev0

Best regards,
Alexandre

**Comment by TheodoreGalanos** [comment]
Hmm, interesting it works for me. This feels like a mismatch happening between different versions of the model? I can load the pytorch model with this code (only having the config.json in the folder).

**Comment by alexrame** [comment]
Is ```model_path = Path("architext/gptj-162M")``` the correct pytorch model to be used here https://github.com/CarperAI/ArchitextRL/blob/2673c676e7ded74818a79b6372bf7cdfbd012049/gradio_UI.py#L12?

**Comment by TheodoreGalanos** [comment]
Oh, yeah that should be the local folder where you downloaded the model (from HF).

I've messed up something in the HF upload, will try and fix it as soon as I can, but in the meantime downloading the model locally should work!

**Comment by alexrame** [comment]
Thanks for the answer. If the HF version is not adequate, how/from where should we download the good model?

**Comment by TheodoreGalanos** [comment]
You can download it from huggingface actually directly: https://huggingface.co/architext/gptj-162M/blob/main/pytorch_model.bin

**Comment by alexrame** [comment]
I am sorry but I don't understand your answers: 

> I've messed up something in the HF upload, will try and fix it as soon as I can, but in the meantime downloading the model locally should work!

If something is wrong in the HF upload, then we should not load the HF model. Yet, you then reply:

> You can download it from huggingface actually directly: https://huggingface.co/architext/gptj-162M/blob/main/pytorch_model.bin

In brief, could you please release a list of instructions to make your notebook usable ?

Best, and sorry for the issue.

**Comment by honglu2875** [comment]
> I am sorry but I don't understand your answers:
> 
> > I've messed up something in the HF upload, will try and fix it as soon as I can, but in the meantime downloading the model locally should work!
> 
> If something is wrong in the HF upload, then we should not load the HF model. Yet, you then reply:
> 
> > You can download it from huggingface actually directly: https://huggingface.co/architext/gptj-162M/blob/main/pytorch_model.bin
> 
> In brief, could you please release a list of instructions to make your notebook usable ?
> 
> Best, and sorry for the issue.

I got the same problem but it worked when I downgrade transformers just slightly. I suspect something went wrong with the newer version of transformer that messed up with GPT-J

And by the way, my effort now has been to focus on streamlit UI because it's simply better in all kinds of ways (from its mechanism to the flexibility).

**Comment by alexrame** [comment]
Thanks for your reply. Could you please share which version you use know?

**Comment by honglu2875** [comment]
> Thanks for your reply. Could you please share which version you use know?

just checked and it's 4.25.1
but I remember 4.27 also works but forgot
